src/components/ui/LogoHeader/LogoHeader.jsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import AnimatedNumbers from "react-animated-numbers";
+import * as S from "./LogoHeader.styles";
+
+export default function LogoHeader({ points = 0 }) {
+  return (
```

Comment:
By default it is 0 then ill give it 200 points, it will change from 0 to 200. What will happen if I give it another 50 points? Will it go from 200 to 250? I Think not. You need to hold state here. So the truth would be state + points